truckersmp_cli/__init__.py

```diff
@@ -492,21 +492,34 @@ def start_with_wine():
     if args.activate_native_d3dcompiler_47:
         activate_native_d3dcompiler_47(args.prefixdir, wine)
 
-    print("""
-    ###################################################################
-    #                                                                 #
-    #  Please check wine steam is running or the launcher won't work  #
-    #                                                                 #
-    ###################################################################
-
-    Press enter if you are good to go: """, end="")
-    sys.stdin.readline()
-
     env = os.environ.copy()
     env["WINEDEBUG"] = "-all"
     env["WINEARCH"] = "win64"
     env["WINEPREFIX"] = args.prefixdir
     env["WINEDLLOVERRIDES"] = "d3d11=;dxgi=" if not args.enable_d3d11 else ""
+
+    print("Waiting for Steam to be running...", end="")
+    sys.stdout.flush()
+    argv = (wine, "winedbg", "--command", "info process")
+    while True:
```

Comment:
Do we need a loop here?
We expect them to have steam already running and a looping check will trigger immediately even when the user isn't logged in yet or steam finds an update and restarts itself (that's why we're listening for loginusers.vdf at proton).